src/lib/containers/entity_finder/tree/TreeView.tsx

```diff
@@ -206,17 +264,59 @@ export const TreeView: React.FunctionComponent<TreeViewProps> = ({
           setDetailsViewConfiguration({
             type: EntityDetailsListDataConfigurationType.USER_FAVORITES,
           })
-
           break
       }
+      setBreadcrumbItems([
+        {
+          name: scope,
+          isCurrent: true,
+          action: () => {
+            setCurrentContainer('root')
+          },
+        },
+      ])
     } else {
       setDetailsViewConfiguration({
         type: EntityDetailsListDataConfigurationType.PARENT_CONTAINER,
         parentContainerId: currentContainer,
       })
+      if (isSuccessBundle) {
+        setBreadcrumbItems([
+          {
+            name: scope,
+            isCurrent: false,
+            action: () => {
+              setCurrentContainer('root')
```

Comment:
The breadcrumbs can control this component, regardless of where they are used:


https://user-images.githubusercontent.com/17580037/114452819-e132a080-9ba6-11eb-9690-9fd257e8d549.mov